newIDE/app/src/ObjectEditor/Editors/SimpleTileMapEditor.js

```diff
@@ -163,12 +163,10 @@ const SimpleTileMapEditor = ({
         height: e.currentTarget.naturalHeight,
       };
       setLoadedAtlasImageDimensions(newDimensions);
-      if (previousAtlasImageResourceName.current === atlasResourceName) {
-        // The resource did not change, do nothing.
```

Comment:
why was this check done before? Was it to prevent unnecessary updates because the function was called a lot? 
Ot just out of safety?